### PR TITLE
es-mx: Using v1.2.1 of covid-19.po and v1.0.1 of ebola.po file

### PR DESCRIPTION
**Pull request by Delizald**

### CODE REVIEW

File: Localization/es-MX/Diseases/kidney-cancer.po [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/kidney-cancer.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/parkinson's-disease.po [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/parkinson's-disease.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/epigentics.po [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/epigentics.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/ebola-virus.po [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0.1\n"
+"Source-Last-Modified: 20201010\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201010\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/ebola-virus.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/dengue-fever.po [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/dengue-fever.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/P53.po [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/P53.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/chagas-disease.po [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/chagas-disease.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/hepatitis-c.po [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/hepatitis-c.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/huntington's-disease.po [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/huntington's-disease.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/zika-virus.po [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20201005\n"
+"Target-Language: es-MX\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20201005\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+Language: es_MX\n"
```
Comment (tematim) [inline Localization/es-MX/Diseases/zika-virus.po]: Remove this line to allow the merge or add double quotes before the word Language

File: Localization/es-MX/Diseases/kidney-cancer.po (hunk 1 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/kidney-cancer.po]: This is completed.

File: Localization/es-MX/Diseases/parkinson's-disease.po (hunk 2 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/parkinson's-disease.po]: This is completed.

File: Localization/es-MX/Diseases/epigentics.po (hunk 3 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/epigentics.po]: This is completed.

File: Localization/es-MX/Diseases/ebola-virus.po (hunk 4 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/ebola-virus.po]: This is completed.

File: Localization/es-MX/Diseases/dengue-fever.po (hunk 5 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/dengue-fever.po]: This is completed.

File: Localization/es-MX/Diseases/P53.po (hunk 6 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/P53.po]: This is completed.

File: Localization/es-MX/Diseases/hepatitis-c.po (hunk 8 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/hepatitis-c.po]: This is completed.

File: Localization/es-MX/Diseases/huntington's-disease.po (hunk 9 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/huntington's-disease.po]: This is completed.

File: Localization/es-MX/Diseases/zika-virus.po (hunk 10 above)
Comment (Delizald) [reply Localization/es-MX/Diseases/zika-virus.po]: This is completed.

**Review by meatball133 — DISMISSED** [review]

**Comment by meatball133** [comment]
Looks good for me, really good job!

**Review by meatball133 — APPROVED** [review]

**Pull request merged**